src/components/templates/ArtPiece/ArtPiece.scss

```diff
@@ -10,6 +10,16 @@ $chat-wrapper-height: 260px;
   height: 100%;
   padding: 0;
   margin: 0;
+  background-attachment: fixed;
+  background-position: center;
+  background-size: cover;
+
+  &__background {
```

Comment:
shouldn't it be in the `ContainerWithBackground` component?

Reply:
fixed b396ea2